assets/sass/components/user-input/googlesitekit-user-input-preview.scss

```diff
@@ -54,15 +54,20 @@
 	.googlesitekit-user-input__preview-answer {
 		align-items: center;
 		background-color: $c-solitude;
-		border-radius: 100px;
+		border-radius: 30px;
 		color: $c-denim;
 		display: flex;
 		font-weight: $fw-secondary-medium;
 		margin-bottom: 12px;
 		margin-right: 12px;
 		padding: 8px 13px;
+		word-break: break-word;
 
 		@include responsive-text("body1", "body3");
+
+		@media (min-width: $bp-tablet) {
+			border-radius: 100px;
+		}
```

Comment:
With a multiline answer for example, this still looks weird. How about always using `30px`, even on wider viewports? IMO this always looks better than `100px` - for one-line content, there's no visual difference anyway (since the height of the bubble then isn't even close to 100 pixels), for multi-line content, using 100 pixels makes the border way too round IMO and leaves little visual spacing around the words.

cc @asvinb Please double-check, since you wrote the IB.

Reply:
Ok, updated. Now it should always be 30px.